### PR TITLE
infra: Don't run scheduled tests on forks

### PR DESCRIPTION
**Pull request by VladimirSlavik**
SSIA.

### CODE REVIEW

**Review by rvykydal — APPROVED** [review]
Thank you!

File: .github/workflows/webui-periodic.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# ======================================
+    # ======================================
```
Comment (vojtechtrefny) [inline .github/workflows/webui-periodic.yml:1]: This looks like an accidental change?

**Review by vojtechtrefny — APPROVED** [review]
LGTM

**Review by jkonecny12 — APPROVED** [review]
This is nice improvement. Ideally we shouldn't have to care because forks should have disabled actions until you need to work with them in the settings. However, in any case, this won't hurt.

**Comment by VladimirSlavik** [comment]
Indeed, ideally it would be like that. Two things break that ideal world:
- If you test actions on your fork, you must enable them globally. Then you have to disable them one by one, except the tested one. But new ones (or renamed) start as enabled anyway. So it's pretty intense in terms of keeping it all under control.
- I'm getting notifications about actions failing on dcantrell's fork. Can't do anything about it...

File: .github/workflows/webui-periodic.yml (hunk 1 above)
Comment (VladimirSlavik) [reply .github/workflows/webui-periodic.yml:1]: Yes, something did not work right :-/ That also fails the test.

**Pull request merged**